backend/src/apiserver/main.go

```diff
@@ -239,6 +273,50 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 	glog.Info("Http Proxy started")
 }
 
+func startWebhook(controllerClient ctrlclient.Client, wg *sync.WaitGroup) *http.Server {
+	glog.Info("Starting the Kubernetes webhooks...")
+
+	tlsCertPath := *webhookTLSCertPath
+	tlsKeyPath := *webhookTLSKeyPath
+
+	topMux := mux.NewRouter()
+
+	pvValidateWebhook, err := webhook.NewPipelineVersionWebhook(controllerClient)
+	if err != nil {
+		glog.Fatalf("Failed to instantiate the Kubernetes webhook: %v", err)
```

Comment:
Is there a reason to not return an error here instead and let the calling function `main` exit?